domains/theublocked.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "jOaawd",
+        "email": "jorti28@apps.pusd.org"
+    },
+    "record": {
+        "TXT": "google-site-verification=6gmt5zyihimbubddc8hei7ysebshvsvotvt0rpfhk24"
```

Comment:
Google site verification is not supported (use the HTML method)

Reply:
Ah My bad!
One quick Question, how do I use the HTML Method.